### PR TITLE
issue #SB-341 feat:Portal frontend telemetry implementation

### PR DESCRIPTION
**Pull request by Aprajita5**
Please review PR and merge.

### CODE REVIEW

File: src/app/thirdparty/libs/angular-inview.js [hunk 1]
```diff
@@ -0,0 +1,387 @@
+// # Angular-Inview
```
Comment (harishgilimi) [inline src/app/thirdparty/libs/angular-inview.js]: should not commit this file take from bower components
use **bower install angular-inview**

**Comment by harishgilimi** [comment]
@Aprajita5 check the telemetry for player and editors with configurations

File: src/app/thirdparty/libs/angular-inview.js (hunk 1 above)
Comment (Aprajita5) [reply src/app/thirdparty/libs/angular-inview.js]: I had used bower install angular-inview but the problem is with one arrow function which is there in angular-inview.js file which was throwing error then we make changes there and move this file in lib.

File: src/app/private/scripts/controllers/announcement/composeAnnouncementController.js [hunk 2]
```diff
@@ -194,7 +194,7 @@ angular.module('playerApp').controller('composeAnnouncementCtrl', ['$rootScope',
          * @memberOf Controllers.composeAnnouncementCtrl
          */
     composeAnn.saveAnnouncement = function () {
-      var url = ''
+      var url = '' // eslint-disable-line no-unused-vars
```
[the remark this answers is not in the document]
Comment (Aprajita5) [reply src/app/private/scripts/controllers/announcement/composeAnnouncementController.js]: we can not remove this variable, after initialising here we are using further. while running eslint its throwing error so i have add that disable-line.

[the remark this answers is not in the document]
Comment (harishgilimi) [reply src/app/private/scripts/controllers/announcement/composeAnnouncementController.js]: I am merging code but you need to fix it

**Pull request merged**